src/index.js

```diff
@@ -0,0 +1,152 @@
+import 'lodash';
+import './style.css';
+import removeTodo from './ui.js';
+
+// list class
+
+// variables
+const checked = 'fa-check-square';
+const unchecked = 'fa-square-o';
+const lineThrough = 'text-decoration-line-through';
+const list = document.querySelector('#list-items');
+let id;
+let LIST = JSON.parse(localStorage.getItem('TODO')) || [];
+
+class TodoList {
+  // add todo list
+  static addToDoList(todo, completed, id) {
+    const done = completed ? checked : unchecked;
+    const line = completed ? lineThrough : '';
+    const item = `
+    <li class=" icon-check card-header d-flex justify-content-between" id="${id}">
+        <div class="text1">
+          <i class="complete fa ${done} " id="${id}" aria-hidden="true"></i>
+          <p class="text ${line}" ${
+  !todo.completed ? 'contenteditable' : ''
+}>${todo}</p>
+        </div>
+        <i class="delete fa-solid fa-ellipsis-vertical" id="${id}"></i>
+      </li>
+    `;
+    const position = 'beforeend';
+
+    list.insertAdjacentHTML(position, item);
+  }
+}
+
+const updateTodo = (todoId, el) => {
+  const todo = LIST.find((todo) => todo.id === parseInt(todoId, 10));
+  if (el.hasAttribute('contenteditable')) {
+    todo.name = el.textContent;
+  }
+};
+
+// add to list
+document.addEventListener('keyup', (e) => {
+  // LIST.sort((a, b) => b.id - a.id);
```

Comment:
Please remove the commented codes, it makes the code ugly.

Reply:
Hi Angom, thanks for noting that